<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>178Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,1">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>下载</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="my_label">
           <property name="text">
            <string>地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="my_url_edit"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="my_label2">
           <property name="text">
            <string>file：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="my_file_edit"/>
         </item>
         <item>
          <widget class="QPushButton" name="my_file_button">
           <property name="text">
            <string>select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="my_label2">
           <property name="text">
            <string>vol：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="my_vol_edit"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="my_start_button">
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    
   </layout>
  </widget>

  <action name="quit_menuitem">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>